metadata/src/main/java/org/apache/kafka/image/loader/MetadataLoader.java

```diff
@@ -212,38 +216,67 @@ private MetadataLoader(
         this.uninitializedPublishers = new LinkedHashMap<>();
         this.publishers = new LinkedHashMap<>();
         this.image = MetadataImage.EMPTY;
-        this.eventQueue = new KafkaEventQueue(time, logContext,
+        this.eventQueue = new KafkaEventQueue(Time.SYSTEM, logContext,
```

Comment:
Is this change related to test?

Reply:
Using mock time doesn't really work with deferred event queue stuff at the moment. I do want to fix this. But right now, it would make `MockTime` unusable to pass it to `KafkaEventQueue`.

I will add a time accessor to `KafkaEventQueue` so it is at least clear when we're using the event queue time